JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/index.js

```diff
@@ -1,4 +1,9 @@
 $(() => {
+  const updateOutput = (value, valueType) => {
```

Comment:
Maybe it's better to name it `formatOutput`? Actual name is too common for me.
Please refactor in for all approaches

Reply:
How about `prettierFormat` instead?